public/locales/en-US/send.ftl

```diff
@@ -7,8 +7,8 @@ uploadPageExplainer = Send files through a safe, private, and encrypted link tha
 uploadPageLearnMore = Learn more
 uploadPageDropMessage = Drop your file here to start uploading
 uploadPageSizeMessage = For the most reliable operation, it’s best to keep your file under 1GB
-uploadPageBrowseButton = Select a file on your computer
-    .title = Select a file on your computer
+uploadPageBrowseButton = Select a file to upload
+    .title = Select a file to upload
```

Comment:
This is the process to change existing strings.
https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_content_best_practices#Changing_existing_strings

You would need a new string ID.